doc/index.md

```diff
@@ -24,3 +24,7 @@
 ### Releases
 
 For releases and release notes see the [Releases](https://github.com/yetanalytics/lrsql/releases) page on the SQL LRS GitHub repository.
+
+### Demos
+
+- [Visualization with Apache Superset](superset.md)
```

Comment:
Don't forget to add this to the front-page README

Reply:
👍 https://github.com/yetanalytics/lrsql/pull/231/commits/3eec709f9f3b36eb03cb1cc067f10613753cfbb9